specs/protocol/differences-from-optimism.md

```diff
@@ -29,9 +29,9 @@
 
 There are two types of network participants in the OP Stack:
 
-- [Sequencers](https://github.com/ethereum-optimism/optimism/blob/develop/specs/introduction.md#sequencers) consolidate
+- [Sequencers](https://specs.optimism.io/introduction.html#sequencers) consolidate
```

Comment:
```suggestion
- [Sequencers](https://specs.kroma.network/introduction.html#sequencers) consolidate
```

Reply:
Since it's a reference to the OP stack, wouldn't the optimism link be correct?